src/config/passport.js

```diff
@@ -0,0 +1,45 @@
+import passport from 'passport';
```

Comment:
Please place all passport strategies in a file that can be named `socialAuthStrategies.js`. This file should be in the Utils folder. Also, ensure your branch is updated with the latest `develop` branch by rebasing.